cypress/support/page-objects/key-presses.po.ts

```diff
@@ -0,0 +1,30 @@
+export class KeyPressesPage {
+  // Page Details
+  static details: PageDetails = {
+    linkText: 'Key Presses',
+    pageUrl: 'key_presses'
+  };
+
+  // Page Selectors
+  private static selectors = {
+    inputElement: '#target',
+    keyNotification: '#result'
+  };
+
+  // Page Functions
+  static visit() {
+    return cy.visit(this.details.pageUrl);
+  }
+
+  static enterKey(key: string) {
+    cy.get(this.selectors.inputElement).type(key);
```

Comment:
Should return these for two reasons
- Allow for extended use of elements, if ever needed in the future
- Keep consistency with other static commands using cy

Reply:
Done